.github/workflows/swift.yml

```diff
@@ -0,0 +1,19 @@
+name: Swift
```

Comment:
Very neat! Would it be possible to skip tests if we don't have xcode 12.2 yet? 
or at least print the xcode version to verify the failure.